js-library/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './constants'
+export * from './services/ens'
+export * from './model'
```

Comment:
I would refactor this in order to reach main functionality of the library more straightforward, e.g. `ENS` while other export are under a distinct object key like `export * as Constants from './constants'`.

When you write the `Usage` examples you will feel